redis/commands/cluster.py

```diff
@@ -644,6 +644,16 @@ def cluster_links(self, target_node: "TargetNodesT") -> ResponseT:
         """
         return self.execute_command("CLUSTER LINKS", target_nodes=target_node)
 
+    def cluster_flushslots(self, target_nodes: Optional["TargetNodesT"] = None) -> None:
+        raise NotImplementedError(
+            "CLUSTER FLUSHSLOTS is intentionally not implemented in the client."
```

Comment:
Would it be better to move this functionality into client rather than specify it for each command? So it will raise an exception whenever any not implemented command will be called

Reply:
Our standard is to do this here... in python at least, we share this way.